auth/src/test/java/com/firebase/ui/auth/testhelpers/TestHelper.java

```diff
@@ -81,4 +83,26 @@ public static void verifySmartLockSave(String providerId, String email, String p
         assertEquals(password, passwordCaptor.getValue());
         assertEquals(providerId, idpResponseCaptor.getValue().getProviderType());
     }
+
+    public static void verifySmartLockSave(String providerId, String email, String phoneNumber,
```

Comment:
Could we fold this into the method above? I know this is a test, but still the method below is almost unreadable. We could just pass in null for the `phoneNumber` or have a simple overload that does that for us.